packages/app-desktop/InteropServiceHelper.ts

```diff
@@ -112,9 +112,11 @@ export default class InteropServiceHelper {
 							// fails on Linux (even if run in the main process).
 							// As such, we use window.print(), which seems to work.
 							//
-							// 2025-05-03: Windows also needs the window.print() workaround.
+							// 2025-05-03: Windows and MacOS also need the window.print() workaround.
+							// See https://github.com/electron/electron/pull/46937.
 
-							if (shim.isLinux() || shim.isWindows()) {
+							const applyWorkaround = true;
+							if (applyWorkaround) {
```

Comment:
It may be possible to restore the original logic after the changes from https://github.com/electron/electron/pull/46937 are included in Joplin.